packages/office-ui-fabric-react/src/components/Modal/docs/ModalA11y.md

```diff
@@ -0,0 +1,3 @@
+- Use `aria-labelledby` and `aria-describedby` on modal container to associate the title and description.
+- Use aria-modal="true" on the modal container.
```

Comment:
```suggestion
- Use `aria-modal={true}` on the modal container.
```

Reply:
this is the trickiness where the accessibility guidance is supposed to be generic to an HTML modal....not specifically 'react'.

Reply:
Hmm okay. Sometimes to be generic I'll leave out *either* quotes or brackets and just say `aria-modal=true`, but I don't know if that's actually better/clearer.